### PR TITLE
fix: select and button the same size

### PR DESCRIPTION
**Pull request by enguerran**
I guess @GoOz will have a better solution for such a problem. But I am not familiar with side-effects on changing something in [cozy-ui](https://github.com/cozy/cozy-ui).

Then, before:

and after:

### CODE REVIEW

**Comment by enguerran** [comment]
[(1) [PHO] Pouvoir choisir la permission de partage](https://trello.com/c/tYuan4nR/399-1-pho-pouvoir-choisir-la-permission-de-partage)

**Comment by enguerran** [comment]
@y-lohse the `padding-bottom` fixes the text which is truncated by the `height` rule.

**Comment by GoOz** [comment]
Well… the issue here is that form elements like `select` have a higher height than button's (in UI I mean) but this mockup introduces a `select` that has a button's height. That's new. And as this is only in this case for now, it'll do the job only on this repo.
But I think there's something to be done in order to get some harmony between all those elements. At some point at least.

tl;dr that's fine by me.
